resources/elasticsearchservice-domain.go

```diff
@@ -42,6 +45,14 @@ func (f *ESDomain) Remove() error {
 
 	return err
 }
+func (e *ESDomain) Properties() types.Properties {
+	properties := types.NewProperties()
+	for _, tagValue := range e.tags {
+		properties.SetTag(tagValue.Key, tagValue.Value)
+	}
+	return properties
```

Comment:
Please also add the `DomainName` as a property.

Reply:
Hello @der-eismann I have committed the changes I used the ListTags function but I am getting the error unknown field 'Resource' in struct literal of type elasticsearchservice.ListTagsInput. Would you be able to help me out with this?

Reply:
Because as you can see in the [docs](https://docs.aws.amazon.com/sdk-for-go/api/service/elasticsearchservice/#ListTagsInput) the struct has no `Resource` field. Instead you need to use the ARN of the resource.